package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_incorrect_cartoon_chartacter",
+  "version": "1.0.0",
+  "description": "PROJECT DESCRIPTION",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Update the test script in the package.json file to run actual tests instead of just echoing an error message. [important]